recipe/meta.yaml

```
@@ -25,8 +25,14 @@ requirements:
   run:
     - python
     - libpython-static  # [not win]
-    - ordered-set >=4.1.0
-    - zstandard >=0.15
+    - ordered-set >=4.1.0  # [py>=37]
+    - orderedset >=2.0.3  # [py<37]
+    - zstandard >=0.15  # [py>=35]
+    - jinja2 >=2.10.2
+    # Optional but recommended
+    - appdirs
+    - tqdm  # [py==27 or py>=34]
+    - pyyaml  # [py==27 or py>=36]
```

Are these selectors necessary? None of these selectors will be useful since we don't support anything lower than 3.9 at the moment? 
```suggestion
    - ordered-set >=4.1.0
    - orderedset >=2.0.3
    - zstandard >=0.15 
    - jinja2 >=2.10.2
    # Optional but recommended
    - appdirs
    - tqdm
    - pyyaml 
```